scripts/tf_cnn_benchmarks/benchmark_cnn.py

```diff
@@ -2183,7 +2183,9 @@ def _build_model_single_session_with_dataset_prefetching(self):
       # Build the per-worker image processing
       function_buffering_resources = data_utils.build_prefetch_image_processing(
           self.model.get_image_size(), self.model.get_image_size(),
-          self.batch_size // len(self.devices), self.cpu_device, self.params,
+          self.batch_size // len(self.devices),
```

Comment:
Thanks for this! Can you add a TODO to add a test? Something like `# TODO(reedwm): Add a test case for this function`. 

Alternatively, you can add the test yourself. The test should be similar to [this test](https://github.com/tensorflow/benchmarks/blob/c12839f9d8389f45c0c4cde92d5ff24588ad35ab/scripts/tf_cnn_benchmarks/benchmark_cnn_distributed_test.py#L314) except it uses real data in the same way that [this test](https://github.com/tensorflow/benchmarks/blob/c12839f9d8389f45c0c4cde92d5ff24588ad35ab/scripts/tf_cnn_benchmarks/benchmark_cnn_distributed_test.py#L406) does.

Reply:
I have added todo for you, I'm sorry that I'm not familiar with this project's test case.